NEWS.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+## [7.5.0] - 2016-08-03
```

Comment:
Also, add a link comparing 7.4.0 to 7.5.0 like the other versions (including 7.4.0 which I just fixed in c4809e109bd467bd6ceaf7b07ee3cd3f962e74c2)